### ISSUE TITLE
[android] catch Java.Net.ConnectException and re-throw a HttpRequestException in SecureHttpClientHandler.cs

### ISSUE THREAD

**Issue by gsgou** [tranb3r/securehttpclient#16]
When there is no internet connection a Java.Net.ConnectException is thrown. Could you add a catch also for it:
https://github.com/tranb3r/securehttpclient/blob/master/SecureHttpClient/Platforms/Android/SecureHttpClientHandler.cs#L196

**Comment by tranb3r** [comment]
Java.Net.ConnectException is a Java.IO.IOException, so it should already be catched and transformed into HttpRequestException here:
https://github.com/tranb3r/securehttpclient/blob/master/SecureHttpClient/Platforms/Android/SecureHttpClientHandler.cs#L204

Based on what you're doing, could you please describe a bit more the expected behavior and the actual behavior ?

**Comment by gsgou** [comment]
You are right: https://docs.oracle.com/javase/7/docs/api/java/net/ConnectException.html
I see it in AppCenter and is not coming over unhandled / unobserved exceptions. Its
catched from ReactiveUIs default exception handler from some observable. I am not quite
sure why i dont see the HttpRequestException as i should. I ll look at it at some point and report.
Thanks for the quick response.

**Comment by tranb3r** [comment]
Maybe it's caused by a request that is NOT sent using SecureHttpHandler ?

**Comment by gsgou** [comment]
That was a good question! Indeed in that old project for this API only the AndroidClientHandler was used instead.
You can close this issue.
